settings-template.jsonc

```diff
@@ -140,5 +140,20 @@
 			"title_single": "One ticket blabla in the last 15 minutes!"
 		}
 		// ... you can add more feeds here
+	],
+
+	"version_feed_interval": 900000,
+
+	"version_feeds": [
+		{
+			// The project whose versions should be monitored
+			"project": "<Project key>",
```

Comment:
I wonder if it might make more sense to just monitor all projects in one feed.
Releses don't happen that often, and I think in #news it works pretty well.

Reply:
I thought it would fit best in the `#java-fixes` channel. I'm not sure how other projects do versions. For example on MCD, versions often only get added after the version has been released. Not as exciting.

But apart from that, this is a technical limitation, JIRA's API only allows for monitoring the versions for only one specific project at once since versions are bound to a project. But if we want to have a feed for version changes in multiple (or all) projects, we could add multiple feeds (even if they all point to the same channel).